src/store.tsx

```diff
@@ -58,7 +59,21 @@ const PokemonContext = createContext({
   deck$,
 });
 
-export const usePokemon = () => useContext(PokemonContext);
+export const usePokemon = () => {
+  const { pokemon$ } = useContext(PokemonContext);
+  return [useObservableState(pokemon$, [])];
+};
+
+export const useDeck = () => {
+  const { deck$ } = useContext(PokemonContext);
+  return [useObservableState(deck$, [])];
+};
+
+export const useSelected = (): [number[], (value: number[]) => void] => {
+  const { selected$ } = useContext(PokemonContext);
+  const write = (value: number[]) => selected$.next(value);
```

Comment:
You could actually change this to a `toggleSelected` and make the logic in the JSX even simpler.

Reply:
Great, now the code look better.